app/js/components/App.jsx

```diff
@@ -62,11 +62,11 @@ class App extends Component {
         <div id="body-wrapper" 
           style={{display :table}}>
           <div id="displayTable"  className="col-md-12 section">
-              <CohortTable
-                toDisplay = {history}
-                description = {description}
-                back={this.back}
```

Comment:
Is this just formatting?

Reply:
Yes, it is. It's one of the issues hound-ci reported

Reply:
We generally prefer formatting changes to have a separate pull request from the one with the actual fix.

Reply:
Ok. I'll split it into two PR but that means one of the PR will come with hound-ci violations

Reply:
Ignore it for today. I will just merge it. But next time, you know what to do in such cases. 😊

Reply:
Ok. Thanks for informing about separating the PR. I'll apply it from now on